### ISSUE TITLE
Why the Generator loss is smaller than the Discriminator loss at the very first epoch? Is it normal?

### ISSUE THREAD

**Issue by ammar7971** [DLangerr/dcgan-mnist-tensorflow#2]
This is one of the  most beautiful, elegant and simple DCGAN implementations i have seen till now Thanks. I have few questions though i will be grateful if you can answer.
1) The theory and even the implementation of GAN tells that you train D first and then G. G generates totally random noise in the first few hundred epochs. So, when i use your implementation the losses at the very start for few hundred epochs oppose the theory i.e. G loss is small and D loss is bigger why? Is it possible that G will have small loss than D at the beginning? 
2) What these losses actually mean for example D loss 1.223344224 and G loss 0.8933454545 ?  Does it mean D did 12% bad and G did 8.9% bad?  (This may sound stupid but i am little confused here)﻿
